crates/holochain_state/src/source_chain.rs

```diff
@@ -510,6 +516,12 @@ impl SourceChain {
             let entries = scratch.drain_entries().collect::<Vec<_>>();
             SourceChainResult::Ok((headers, ops, entries))
         })?;
+        let mut ops_to_self_publish = HashSet::with_capacity(ops.len());
```

Comment:
Is this more accurately "ops to integrate", or does it make sense to think of it as self-publishing? It's not immediately clear how setting the validation stage to "awaiting integration" leads to self-publishing.

```suggestion
        let mut ops_to_integrate = HashSet::with_capacity(ops.len());
```

Reply:
@freesig can you weigh in on this?

Reply:
Yeh I've never thought of integration as including validation. It's a separate workflow after all.

Reply:
@freesig ok then would my suggestion make sense then? Looks like these are the ops to integrate, since there's no need to validate them. Maybe if self publishing were a thing this would also make sense, but like you said, it's not.